core/src/main/java/com/redhat/lightblue/client/LightblueClientConfiguration.java

```diff
@@ -6,6 +6,21 @@
 
 public class LightblueClientConfiguration {
 
+    public enum Compression {
+        NONE, LZF;
+
+        public static Compression parseCompression(String str) {
+            switch (str) {
+            case "NONE":
+                return NONE;
+            case "LZF":
+                return LZF;
+            default:
+                throw new IllegalArgumentException("Invalid compression " + str + ". Supported values are: NONE, LZF");
```

Comment:
I'm not sure if you can configure that server side, but if the server does not flag the response as compressed with a special header, then the client will not try to decompress it. It works the same the other way around - if the client does not set a special header to indicate it's capable to decompress lzf, lightblue will not compress responses.